docs/2101UserGuide.md

```diff
@@ -435,6 +435,19 @@ Here is an example command where we find all members with the names of "Anthony"
 
 `findMember anthony nicholas`
 
+<div markdown="block" class="alert alert-info">:information_source:
+The `findMember` command requires the names to be typed in full, meaning "Nich" will not suffice for finding "Nicholas". 
+The names do not have to be capitalised too! 
+</div>
+
+<div markdown="block" class="alert alert-primary">:bulb:
+The find commands are powerful tools to help you work faster. Want to view the enrolment of Nicholas, but you don’t 
+want to scroll the member list? Use the `findMember` then follow it with the `viewMember` command to achieve that. 
+
+**This is applicable for both members and events**
+
+</div>
```

Comment:
Hmm Is it weird to have it in between? For me, it's either we put it in front or at the end